tasks/main.yml

```diff
@@ -25,6 +33,14 @@
 - name: Create data directory
   file: path={{ otp_data_dir }} mode=0775 state=directory
 
-- name: Install upstart service
-  template: src=otp.conf.j2 dest=/etc/init/otp.conf
+- name: Install service config
+  template: src=otp.service.j2 dest=/lib/systemd/system/otp.service
   notify: Restart OpenTripPlanner
+  register: otp_service_config
+
+  # We need to reload systemd config for it to see changes.  This (and the 'register' just above)
+  # will not be necessary in Ansible 2.2 because there will be a 'systemd' module with a
+  # 'daemon_reload' option that we can use instead of 'service'.
+- name: Notify systemd of service config changes
```

Comment:
I think we can use the `systemd` module here explicitly to get the desired effect (`daemon_reload`), unless that is broken too: https://docs.ansible.com/ansible/systemd_module.html

Regardless, it would be good to get a reference to that issue so that I can track it.